views/message.pug

```diff
@@ -0,0 +1,20 @@
+extends layout
+
+block content
+  .container
+    if messages.success
+      div.text-success
+        for success in messages.success
+          div= success.msg
+    if messages.error
+      div.text-danger
+        for error in messages.error
+          div= error.msg
+
+    h3 Call Rest API Form
+
+    form(method='POST')
```

Comment:
@marciodaitan you don't need to setup the [`action` attribute](http://www.w3schools.com/tags/att_form_action.asp) for this form?